src/test/java/redis/clients/jedis/JedisClusterTest.java

```diff
@@ -741,6 +741,65 @@ public void clusterRefreshNodes() throws Exception {
     }
   }
 
+  @Test
```

Comment:
We should put a timeout here as a long duration is used in the test.
```suggestion
  @Test(timeout = 300_000)
```

Reply:
Modified topologyRefreshPeriod to 1s and the timeout of the entire test to 30s